bld/build-namelist

```diff
@@ -927,6 +929,9 @@ if ($prognostic_mode eq 'prescribed') {
         if ($SSTICE_DATA_FILENAME eq 'UNSET') {
                 die "SSTICE_DATA_FILENAME must be set for MPAS-Seaice prescribed mode \n";
         }
+        if ($SSTICE_MESH_FILENAME eq 'UNSET') {
+                die "SSTICE_MESH_FILENAME must be set for MPAS-Seaice prescribed mode \n";
+        }
```

Comment:
These 3 lines seem to be repeated to me. We can probably remove them.

Reply:
I don't see the repeat.

Reply:
Sorry my eyes were broken `...DATA...` vs `...MESH...`